src/components/buttons/buttons.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import modifiers from '../../modifiers';
+
+const Buttons = ({
+  children,
+  className,
+  hasAddons,
+  position,
+  ...allProps
+}) => (
+  <div
+    {...modifiers.clean(allProps)}
+    className={classnames('buttons', className, modifiers.classnames(allProps), {
+      'has-addons': hasAddons,
+      [`is-${[position]}`]: position,
+    })}
+  >
+    {children}
+  </div>
+);
+
+Buttons.propTypes = {
+  ...modifiers.propTypes,
+  className: PropTypes.string,
+  hasAddons: PropTypes.bool,
+  position: PropTypes.oneOf([null, 'centered', 'right']),
+};
+
+Buttons.defaultProps = {
+  className: '',
+  hasAddons: false,
```

Comment:
We can use `undefined` instead of `false` for default values so it not displayed on the React Dev Tools and reduce the noise. (I started to change this but not all components are using this approach yet)

Reply:
Will do.